test/integration/instrumentation/http-outbound.tap.js

```diff
@@ -111,6 +111,43 @@ tap.test('external requests', function (t) {
     }
   })
 
+  t.test('should recognize requests via proxy correctly', function (t) {
+    const proxyServer = http.createServer(function onRequest(req, res) {
+      req.resume()
+      res.end('ok')
+    })
+    t.teardown(() => proxyServer.close())
+
+    proxyServer.listen(0)
+
+    helper.runInTransaction(agent, function inTransaction() {
+      const opts = {
+        host: 'localhost',
+        port: proxyServer.address().port,
+        path: 'https://www.google.com/proxy/path',
```

Comment:
Overall, nice test. can you just assert the url _could_ get proxied if you didn't terminate the connection?

```suggestion
  t.test('should recognize requests via proxy correctly', function (t) {
    const proxyUrl = 'https://www.google.com/proxy/path'
    const proxyServer = http.createServer(function onRequest(req, res) {
      t.equal(req.url, proxyUrl)
      req.resume()
      res.end('ok')
    })
    t.teardown(() => proxyServer.close())

    proxyServer.listen(0)

    helper.runInTransaction(agent, function inTransaction() {
      const opts = {
        host: 'localhost',
        port: proxyServer.address().port,
        path: proxyUrl
```

Reply:
Hi, assert with proxyUrl added.
CLA signed.